activate.sh

```diff
@@ -17,8 +17,8 @@ autoenv_init() {
 	# We do this in a subshell so we can cd/chdir
 	_files="`
 		_hadone=''
-		while [ "${PWD}" != "${_mountpoint}" ]; do
-			_file="${PWD}/${AUTOENV_ENV_FILENAME}"
+		while [ "$(pwd -P)" != "${_mountpoint}" ]; do
+			_file="$(pwd -P)/${AUTOENV_ENV_FILENAME}"
```

Comment:
If .env file exists in symlinked directory then it should exists on the original physical(filesystem) too. So I don't see any issue here and I have this code running in my system for almost a month now and have not noticed any issue at least on OSX.